### ISSUE TITLE
Fix double-header handling

### ISSUE THREAD

**Issue by dowski** [dowski/scoreboard#3]
There was a double-header the other day and the second game seemed to have a bogus start time. It actually started much earlier than the "scheduled" start time (around 8:30 I think).

```
2 games today
 Tracking for today's game (2018_05_03_tormlb_clemlb_1) will start at 01:10PM (18599 seconds from now)
 Tracking for today's game (2018_05_03_tormlb_clemlb_2) will start at 11:33PM (55979 seconds from now)
```

**Comment by dowski** [comment]
I did some work on this back in bc1c254. It isn't perfect but some double-headers get properly scheduled.

I'll keep this open in case I spot another failure and can gather some details.

**Comment by dowski** [comment]
Here's an example of a failure from some recent logs:

```
Aug 15 18:05:55 scoreboard2 scoreboard[2426]: Cleveland Guardians: 4, Detroit Tigers: 1, Middle of 9 (b:1, s:0, o:3, runners:())
Aug 15 18:05:55 scoreboard2 scoreboard[2426]: game no longer trackable, status: In Progress
Aug 15 18:20:56 scoreboard2 scoreboard[2426]: 2 games today
Aug 15 18:20:57 scoreboard2 scoreboard[2426]: Cleveland Guardians: 4, Detroit Tigers: 1, Top of 9 (b:1, s:0, o:3, runners:())
Aug 15 18:20:57 scoreboard2 scoreboard[2426]: game no longer trackable, status: Final
Aug 15 18:20:58 scoreboard2 scoreboard[2426]: Cleveland Guardians: 0, Detroit Tigers: 0, top of None (b:0, s:0, o:0, runners:())
Aug 15 18:20:58 scoreboard2 scoreboard[2426]: game no longer trackable, status: Pre-Game
```

Looks like the Pre-Game state needs to be trackable.

**Comment by dowski** [comment]
I think this should be fixed in 7c6cdf6.
